pkg/plugins/golang/deploy-image/v1alpha1/scaffolds/internal/templates/controllers/controller.go

```diff
@@ -221,6 +222,23 @@ func (r *{{ .Resource.Kind }}Reconciler) Reconcile(ctx context.Context, req ctrl
 		return ctrl.Result{Requeue: true}, nil
 	}
 
+	var statusMessage string
+	successStatusMessage := "Deployment %s/%s have correct number of AvailableReplicas which is %s"
+	failStatusMessage := "Deployment %s/%s have incorrect number of AvailableReplicas, have %s but should be %s"
+	if found.Status.AvailableReplicas != size {
+		statusMessage = fmt.Sprintf(failStatusMessage, {{ lower .Resource.Kind }}.Name, {{ lower .Resource.Kind }}.Namespace, found.Status.AvailableReplicas, size)
```

Comment:
`%s/%s` should be `{{ lower .Resource.Kind }}.Namespace, {{ lower .Resource.Kind }}.Name`.

Reply:
closing because I think it changed. But I will ping you for a final review when it be done.